### PR TITLE
Add more context

### PR DESCRIPTION
**Pull request by yorinasub17**
This adds more context to the request for web based airbrakes

### CODE REVIEW

File: django_airbrake/utils/client.py [hunk 1]
```diff
@@ -75,16 +76,15 @@ def _generate_xml(self, exception=None, request=None):
...
                 request.get_full_path())
             etree.SubElement(request_em, 'url').text = str(url)
+            url_el.text = url
 
-            cb,_,_ = resolve(request.path)
+            cb, _, _ = resolve(request.path)
             etree.SubElement(request_em, 'component').text = str(cb.__module__)
             etree.SubElement(request_em, 'action').text = str(cb.__name__)
-
-            if len(request.POST):
```
Comment (phoxelua) [inline django_airbrake/utils/client.py:83]: why are we disabling post?

Comment (yorinasub17) [inline django_airbrake/utils/client.py:83]: This is sending the post args to airbrake, which is a security risk.

**Comment by phoxelua** [comment]
side note, i hear this fork is supposed to be deprecated. is that why we don't do tests?

File: django_airbrake/utils/client.py (hunk 1 above)
Comment (phoxelua) [inline django_airbrake/utils/client.py:83]: oh right that's bad. so how will we debug post args now? via splunk?

**Comment by phoxelua** [comment]
ok you assuaged my doubts offline. lgtm!

**Pull request merged**